src/components/UI/AvailabilityZonesLabel.js

```diff
@@ -86,15 +86,14 @@ function AvailabilityZonesLabel({
     isMaxReached && !isRadioButtons ? 'is-max-reached' : '';
   const classNames = `${letter} ${notCheckedClass} ${pointerClass} ${isMaxReachedClass}`;
 
-  const onClick = () => {
-    isMaxReached && !isChecked && !isRadioButtons
-      ? null
-      : onToggleChecked &&
-        onToggleChecked(!isChecked, { title, letter, label });
+  const toggleChecked = () => {
+    if (!isMaxReached || isChecked || isRadioButtons) {
```

Comment:
Yes, it behaves the same. It just a readibility improvement

Reply:
Are you sure?

I just tried the following in `node`

```js
> let [isMaxReached, isChecked, isRadioButtons] = [true, false, true]
undefined
> Boolean(!isMaxReached || isChecked || isRadioButtons)
true
> Boolean(isMaxReached && !isChecked && !isRadioButtons)
false
```

Reply:
It works as expected because initially, if the `isMaxReached && !isChecked && !isRadioButtons` check was falsy, the function would get executed. Now, if the `!isMaxReached || isChecked || isRadioButtons` check is truthy, the function gets executed.

Reply:
Ok I trust you for this one.. :)